### ISSUE TITLE
opencore 080 in virtualbox give this guru medication error

### ISSUE THREAD

**Issue by kocoman1** [acidanthera/bugtracker#2016]
opencore 080 in virtualbox give this guru medication error

```
00:00:13.441922 Display::i_handleDisplayResize: uScreenId=0 pvVRAM=00007f1b6c000000 w=7680 h=4320 bpp=32 cbLine=0x7800 flags=0x0 origin=0,0 |  
-- | --
00:00:13.582583 Receiving EFI panic... |  
00:00:13.583788 EFI: Panic! EFI Assertion failed! |  
00:00:13.583789 File: r:\tinderbox\efi2-6.1\src\vbox\devices\efi\firmware\VBoxPkg\VBoxAppleSim\VBoxAppleSim.c |  
00:00:13.583790 Line: 135 |  
00:00:13.583790 Description: AppleGetVar_Unknown0: pvArg1=D99EB598 pvArg2=D99EB118 pvArg3=0 pvArg4=DB237318 |  
00:00:13.583817 Changing the VM state from 'RUNNING' to 'GURU_MEDITATION' |  
00:00:13.583829 Console: Machine state changed to 'GuruMeditation' |  
00:00:13.584178 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! |  
00:00:13.584179 !! |  
00:00:13.584179 !! VCPU0: Guru Meditation -225 (VERR_INTERNAL_ERROR) |  
00:00:13.584184 !! |  
00:00:13.584199 !! |  
00:00:13.584199 !! {mappings, <NULL>} |  
00:00:13.584200 !!
```

the picker not shown

**Comment by vit9696** [comment]
This one is a bug on VirtualBox side. We will not guarantee compatibility, but you can try enabling DeviceProperties protocol override.

**Comment by hwertz** [comment]
Just to add, I did what you suggested and it worked!  (went to config.plist, found ProtocolOverrides section, set DeviceProperties to true.  Also changed screen resolution while I was at it.)   I hit escape at bootup so I could pick to boot from the hard drive, VB keeps booting straight into macOS until it's (sucessfully) booted into OpenCore once.

Why OpenCore?  VirtualBox doesn't need OpenCore for Monterey, but I installed KVM-OpenCore (which already has CryptexFixup included) so I can have a go at updating a VM to Ventura on my Ivy Bridge system.
